prow/e2e-simpleTests-cni.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#######################################
+#                                     #
+#         e2e-simpleTest-cni          #
+#                                     #
+#######################################
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+echo 'Running e2e_simple test with rbac, auth Tests and CNI enabled'
+export ENABLE_ISTIO_CNI=true
+# cniBinDir setting is appropriate for GKE environments
+export EXTRA_HELM_SETTINGS="--set istio-cni.excludeNamespaces={} --set istio-cni.cniBinDir=/home/kubernetes/bin --set istio-cni.tag=nightly-master"
+# only gke-e2e-test-latest is enabled for Networkpolicy and Calico
+export RESOURCE_TYPE="gke-e2e-test-latest"
+export E2E_ARGS+=" --kube_inject_configmap=istio-sidecar-injector"
+./prow/e2e-suite.sh --auth_enable --single_test e2e_simple "$@"
```

Comment:
Any reason to set ENABLE_ISTIO_CNI, EXTRA_HELM_SETTINGS, and E2E_ARGS here instead of just use the e2e_simple_cni target you added to istio.mk?

Reply:
Ah yeah... the bin dir delta in GKE.

Reply:
+1 for the first comment, and I don't understand the second. =p

Reply:
The e2e_simple_cni target was added for user ease and referenced in the CNI test readme file.   It is more for local testing than the CI pipeline.   I could use it here, but it doesn't really simplify much, especially since I have additional settings like the bin dir and resource_type to do anyway.   Since the end goal is probably to have most e2e tests run with the CNI enabled I did not want to end up duplicating all those make targets.  As we start using the CNI for all the e2e tests we would just do it in one of the sh wrappers to set the proper environmentals.   IOW not by replicating all the make targets.